tests/issue_1512/randomFiles.py

```diff
@@ -0,0 +1,28 @@
+import string
```

Comment:
FYI, this is a much quicker and simpler way to create files of a certain size in a shell script:
```
$ time head -c $((1024*1024)) /dev/urandom > /tmp/t.dat
r:0m0.012s  u:0m0.001s  s:0m0.011s
```